drivers/input/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ zephyr_library_sources_ifdef(CONFIG_INPUT_KBD_MATRIX input_kbd_matrix.c)
 zephyr_library_sources_ifdef(CONFIG_INPUT_NPCX_KBD input_npcx_kbd.c)
```

Comment:
Please re-format commit title and message as per https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines

Reply:
Ok fixed

Reply:
Done Ok